res/values-en/strings.xml

```diff
@@ -52,7 +52,7 @@
     <string name="pref_reset">Reset</string>
     <string name="pref_reset_summary">Use Default Settings to Replace the contents in /sdcard/rime</string>
 
-    <string name="pref_preferences">Preferences</string>
+    <string name="pref_keyboard">Preferences</string>
```

Comment:
What I did(order and translate) is referring AOSP.
It has similarity with trime in preference(keyborad).

![aosp](https://cloud.githubusercontent.com/assets/7157227/22182583/ab35c804-e0e3-11e6-9dda-aa2199700d0d.png)
![pre](https://cloud.githubusercontent.com/assets/7157227/22182588/b2fa45e2-e0e3-11e6-8b23-36cf5af2d095.png)